### PR TITLE
Bug Fix making CLI work again.

### PR DESCRIPTION
**Pull request by Schultzer**
- Bug fix making CLI work again line 90.
- ~~Refactored walkSync using path.join for better handling.~~
- Fixed verboseDrawContur.
- Syntax fix in `processor_spec.js`

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ function action(paths, options) {
 
   var filesDone = [];
 
-  for (var j = 0; j < files.length; j++) {
+  for (var i = 0; i < files.length; i++) {
```
Comment (danschultzer) [inline cli.js]: Why did you change this instead of just changing all the indexes below?

Comment (Schultzer) [inline cli.js]: Why should we use j here as the only place when we use i everywhere else.?
There are 5 for-loops and 4 of them use i for index and this one we changed to j.

Comment (danschultzer) [inline cli.js]: `var i` is already defined and used.

Comment (Schultzer) [inline cli.js]: we can just remove `var`

Comment (danschultzer) [inline cli.js]: :+1:
